haiku-apps/wordclock/wordclock-1.0.recipe

```diff
@@ -0,0 +1,44 @@
+SUMMARY="An imprecise clock replicant"
+DESCRIPTION="WordClock is imprecise clock replicant and this show clock with word"
+HOMEPAGE="https://github.com/Janus2/WordClock"
+COPYRIGHT="2017 Janus"
+LICENSE="MIT"
+REVISION="1"
+srcGitRev="cbb5da9a498ec99a71a68f039042601207eafdd1"
+SOURCE_URI="https://github.com/Janus2/WordClock/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="fe29a3f17467927f88052d9c260b894d8424cf175b633aa2ad22deea2b25abba"
+SOURCE_FILENAME="WordClock-$srcGitRev.tar.gz"
+SOURCE_DIR="WordClock-$srcGitRev"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+
+PROVIDES="
+	WordClock = $portVersion
```

Comment:
This should be the same low-casing as the recipe name: "wordclock"

Reply:
Good catch. Thanks @humdingerb!

Reply:
@localanu, if you are unsure about what should be converted to lower-case, well, it's only for the first line of PROVIDES, not for the app: line.
So this is what we should have for the PROVIDES:
```
PROVIDES="
	wordclock = $portVersion
	app:WordClock = $portVersion
	"
```

For the DESCRIPTION, @humdingerb's suggestion will look like this:
```
DESCRIPTION="WordClock is an imprecise clock that displays the time in words \
instead of digits. It can be used as a Replicant as well."
```
Hmm, @humdingerb, wouldn't it be better to write "replicant" all lower-case?
(Unless, of course, the Haiku docs mention "Replicant").

@localanu, in BUILD, don't forget to add
```
	make bindcatalogs
```
after the
```
	make $jobArgs
```
as @humdingerb asked.

Also, please don't forget to remove the empty blank line at the end of the recipe.

Thanks!